cibyl/cli/parser.py

```diff
@@ -115,13 +132,10 @@ def parse(self, arguments: List[Argument] = None) -> None:
                          arguments.items() if arg_value is not None and not
                          isinstance(arg_value, Argument)}
 
-    def get_group(self, group_name: str) -> argparse._ArgumentGroup:
+    def get_group(self, group_name: str) -> Optional[argparse._ArgumentGroup]:
         """Returns the argument parser group based on a given group_name
 
         :param group_name: The name of the group
-
-        :return: An argparse argument group if it exists and matches
-                 the given group name, otherwise returns None
```

Comment:
Why would you remove this documentation?

Reply:
That was a mistake, I probably mistook it as a type hint, I'll restore it